### ISSUE TITLE
prop: BUG

### ISSUE THREAD

**Issue by remotejob** [libdns/digitalocean#5]
Example don't work:
`ERROR: POST https://api.digitalocean.com/v2/domains/NYC3/records: 404 The resource you were accessing could not be found.`
It must be `https://api.digitalocean.com/v2/domains/records` 

If ZONE :="/"
`ERROR: POST https://api.digitalocean.com/v2/domains///records: 404 The resource you were accessing could not be found`

**Comment by mholt** [comment]
Possibly this problem? https://github.com/libdns/digitalocean/pull/3

**Comment by remotejob** [comment]
No, I don't think. It's only related to libdns, in  digitalocean case ZONE not used. For my opinion it's very easy to fix.
Hmm.. I decide use LEGO lib. In fact  in begin "CADDY" use LEGOlib for DNS-01.
[https://twitter.com/mholt6/status/1073103805112147968](https://twitter.com/mholt6/status/1073103805112147968)

**Comment by mholt** [comment]
How should we fix it?

(You can read more about why we switched from lego in https://github.com/caddyserver/certmagic/issues/71.)

**Comment by remotejob** [comment]
Fix, maybe somewhere:
if digitalocean {
 not use zone
else {
.....
}
Sorry,  I don't have time exam (certmagic) project.
From other side LEGO support about 50 DNS providers. Why not use it? Or at least give option use it.

**Comment by mholt** [comment]
> From other side LEGO support about 50 DNS providers. Why not use it? Or at least give option use it.

Lego is not a general-purpose DNS library like libdns is. All its providers can do is set and delete ACME TXT records. But if you're using Caddy, you can still use lego, it's just a bit clunky to configure: https://github.com/caddy-dns/lego-deprecated

**Comment by remotejob** [comment]
I see good! Unfortunately it my case digitalocean provider don't work for me. I have about 2000 domains to convert for https:
In fact looks like it simple tupo:
 POST https://api.digitalocean.com/v2/domains/NYC3/records
 POST https://api.digitalocean.com/v2/domains/records
